data/hai/unfettered 0 prologue.txt

```diff
@@ -297,6 +297,127 @@ mission "Unfettered Jump Drive 3"
 
 
 
+mission "Unfettered Jump Drive Trading"
+	minor
+	landing
+	invisible
+	source "Darkcloak"
+	to offer
+		has "Unfettered Jump Drive 3: offered"
+		not "Wanderers: Jump Drive Source: active"
+		# There is no minimum amount of JD to sell (other than the first 3), except if you failed to prove yourself the first time.
+		# "Unfettered Jump Drive 4: failed" represents the amount of JD sold.
+		"unfettered: jd to sell" < "Unfettered Jump Drive 4: failed"
+		or
+			# guaranteed if you sold more than 10 JD
+			"Unfettered Jump Drive 4: failed" > 10
+			random < 25
+			and
+				"Unfettered Jump Drive 4: failed" > 5
+				random < 50
+	on offer
+		conversation
+			`The Unfettered are starting to know you by reputation for bringing jump drives to them. This time, one of them - an engineer, judging by the writing pad and equipment they have - is waiting for you at your hatch. Maybe they mean to trade something other than money with you. Do you want to bargain for another drive with them?`
+			choice
+				`	(No.)`
+					defer
+				`	(Yes.)`
+			`	You exit your ship and immediately the engineer starts to lead you to the outfitter. "We will show you the weapon once you arrive. For now you will only be allowed to access one of them, but if you continue to prove of great assistance to us we may give you access to the others."`
+			`	They lead you through the usual outfitter but then go to an area you never noticed before, probably because you were not meant to. As you are about to pass through what you assumed to be a security check, an Unfettered throws a blade at your face!`
+			choice
+				`	(Try to grab it.)`
+					goto grab
+				`	(Try to dodge it.)`
+					goto dodge
+				`	(Leap to the side and use my own weapon.)`
+				`	(Flee.)`
+					goto flee
+			`	You leap to the side, grabbing your own weapon just in time to threaten the Unfettered that threw the blade at you. Seeming a bit surprised by your unconventional tactics, they lower their own weapon. "Well done! In truth, there is no higher honor than surviving on the field of battle, because only then can you bring loot back to those who need it! A tactic is only good if you can find and execute it quickly enough to give you an edge, which you did. I am Desneeparu, and it is my pleasure to meet you, captain."`
```

Comment:
```suggestion
			`	You leap to the side, grabbing your own weapon just in time to threaten the Unfettered that threw the blade at you. Seeming a bit surprised by your unconventional tactics, they lower their own weapon. "Well done! In truth, there is no higher honor than surviving on the field of battle, because only then can you bring loot back to those who need it! A tactic is only good if you can find and execute it quickly enough to give you an edge, which you did. I am Desneeparu, and it is my pleasure to meet you, Captain."`
```